src/btree/bt_curprev.c

```diff
@@ -553,6 +555,14 @@ __cursor_row_prev(WT_CURSOR_BTREE *cbt, bool newpage, bool restart, size_t *skip
 restart_read_page:
         rip = &page->pg_row[cbt->slot];
         WT_RET(__cursor_row_slot_key_return(cbt, rip, &kpack, &kpack_used));
+        if (F_ISSET(&cbt->iface, WT_CURSTD_PREFIX_SEARCH) && prefix != NULL) {
+            WT_RET(__wt_prefix_match(
+              session, CUR2BT(cbt)->collator, prefix, &cbt->iface.key, &prefix_match));
+            if (prefix_match) {
```

Comment:
The logic or naming here feels wrong to me. If there was a prefix match, then I'd expect to *not* return WT_NOTFOUND.

Reply:
Yeah that was backwards, sorry fixed now.